src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,13 +14,41 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all persons in the address book."
+            + "Parameters: " + "[" + PREFIX_SORT + "FIELD] (must be n, d or g)\n"
```

Comment:
Maybe you can consider adding `N, D, G` here since they are also accepted by the list command.

Reply:
Ah yes, thanks for highlighting this. I forgot to change the message usage to show that upper and lower case letters are accepted. I will also update the UG with this.